### PR TITLE
Create hall model with tests

### PR DESCRIPTION
**Pull request by Joannitka-pl**

### CODE REVIEW

File: spec/models/hall_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Hall, type: :model do
+
+  before(:each) do
+    @hall = Hall.create(number: 1, capacity: 100)
+  end
+
+  it 'creates hall with valid number and capacity' do
```
Comment (wmaciejak) [inline spec/models/hall_spec.rb:9]: Consider using `describe`. For instance:

```ruby
describe "validations" do
  context "when everything is passed" do
    it 'creates hall with valid number and capacity' do   
      expect(@hall).to be_valid  
    end
  end

  context "when there is no number in arguments" do
    ...
  end
end
```

**Comment by wmaciejak** [comment]
Except for one comment, everything is really cool! Good job! 
For the next steps, consider:
1. Adding relationships to currently created models
2. Migrate some controllers to Monte way

**Comment by Joannitka-pl** [comment]
@wmaciejak proszę o CR.

**Comment by wmaciejak** [comment]
Looks really good 👍 If tests are passing, please merge it and start working on fresh branch (and PR) :)

**Review by wmaciejak — APPROVED** [review]

**Pull request merged**